resources/docs/accounts.md

```diff
@@ -0,0 +1,90 @@
+# Accounts
```

Comment:
Please, keep documentation in single `Documentation.md` file.

Also, please note that overview of supported services is already there: https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#supported-feed-formats-and-online-feed-services

Also, note that concepts shared by almost all of services are the same, therefore it is arguably better to only explain not-so-obvious things (the documentation is meant for slightly advanced users) like what is "Application ID" (OAuth 2.0 client ID) or "Application key" (OAuth 2.0 client secret).

Also do not documentate things which are self-explanatory like "Download only unread messages".

Also if you feel that the individual service plugin deserves more in-depth documentation, create 2nd level heading and write your specific documentation stuff under it, like seen: https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#gmail

Reply:
I get OAuth and its terminology but I'm confident people who use RSS Guard might not be familiar with how it operates and might find its terrms inscrutable. Myself I had a bit of a hard time reading through the Google documentation and making sense of it.